frontend/src/App.res

```diff
@@ -22,12 +22,11 @@ fragment BenchmarkMetrics on benchmarks {
 `)
 
 module GetBenchmarks = %graphql(`
-query ($repoId: String!, $pullNumber: Int, $isMaster: Boolean!, $startDate: timestamp!, $endDate: timestamp!, $comparisonLimit: Int!) {
-  benchmarks: benchmarks(where: {_and: [{pull_number: {_eq: $pullNumber}}, {pull_number: {_is_null: $isMaster}}, {repo_id: {_eq: $repoId}}, {run_at: {_gte: $startDate}}, {run_at: {_lt: $endDate}}]}) {
+query ($repoId: String!, $pullNumber: Int, $isMaster: Boolean!, $benchmarkName: String, $isDefaultBenchmark: Boolean!, $startDate: timestamp!, $endDate: timestamp!, $comparisonLimit: Int!) {
+  benchmarks: benchmarks(where: {_and: [{pull_number: {_eq: $pullNumber}}, {pull_number: {_is_null: $isMaster}}, {repo_id: {_eq: $repoId}}, {benchmark_name: {_is_null: $isDefaultBenchmark, _eq: $benchmarkName}}, {run_at: {_gte: $startDate}}, {run_at: {_lt: $endDate}}]}) {
```

Comment:
A few questions:

- The default/main benchmark _must_ be set to `NULL`?
- What happens if all of the benchmarks have a `benchmark_name` (i.e. they're not `NULL`?)

Reply:
Currently yes the default benchmark must be `NULL`. When we first select a repository, the data for the `master` branch and for `NULL` benchmark is fetched. If all benchmarks have a `benchmark_name`, data will be empty:

<img width="1326" alt="Screenshot 2021-03-24 at 08 40 06" src="https://user-images.githubusercontent.com/5595092/112273470-ad3a1e80-8c7d-11eb-8c8e-787bf88ba9b9.png">

As soon as we select a benchmark (`Single` in the screenshot), we'll get some results

Reply:
As you pointed out in https://github.com/ocurrent/current-bench/pull/69#pullrequestreview-618490215, there's currently no way to figure out what's the default benchmark if none of them are `NULL`.